packages/block-editor/src/components/grid-visualizer/grid-item-resizer.js

```diff
@@ -164,8 +165,9 @@ function GridItemResizerInner( {
 					 * isn't directly above the handle, so we try to detect if it happens
 					 * outside the grid and dispatch a mouseup event on the handle.
 					 */
-					const rootElementParent =
-						rootBlockElement.closest( 'body' );
+					const rootElementParent = rootBlockElement.closest(
+						'.editor-styles-wrapper'
+					);
 					rootElementParent.addEventListener(
```

Comment:
Could there ever be a case where `rootElementParent` isn't available? I assume not, and this isn't anything that's changed in this PR so feel free to ignore for now. Longer-term, just wondering if it'd be good to cautiously guard by adding optional chaining, but I'm likely overthinking it 😄

Reply:
I think this is solved with attaching the listener to `ownerDocument`.